src/styles/blocks/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$calendar-width: 720px;
```

Comment:
![image](https://user-images.githubusercontent.com/102512764/201375726-bdd9e8cf-719e-48b2-b015-99209e0a6b53.png)
Actually, the width of the calendar is 740px and therefore it is not centered.

Reply:
Thanks, I centered it.